test/test_nestedtensor.py

```diff
@@ -482,6 +481,7 @@ def test_device_checks(self, device):
         self.assertEqual(nt.is_cuda, is_cuda)
 
     @dtypes(torch.float, torch.float16, torch.double)
+    @torch.inference_mode()
```

Comment:
Why?

Reply:
I think this was a rebase bug good catch. Will remove